djstripe/models.py

```diff
@@ -267,7 +271,7 @@ def subscription(self):
 
     # TODO: Accept a coupon object when coupons are implemented
     def subscribe(self, plan, charge_immediately=True, **kwargs):
-        # Convert Plan to stripe_id
+        """Convert Plan to stripe_id."""
```

Comment:
Hmm what would you like me to do for these instances as https://github.com/PyCQA/pydocstyle/issues/97 is still an issue?